src/content/guides/using-persistent-volumes-with-csi-driver-on-aws/index.md

```diff
@@ -0,0 +1,120 @@
+4. Click the _Configure & Install_ button, select the cluster and it will be installed.
+
+## Storage Classes
+
+Your Kubernetes cluster will have a Storage Class `ebs-csi` deployed, once you installed the `aws-ebs-csi-driver-app` from _Giant Swarm Playground_ catalog.
+
+The storage class offers volume encryption and allows resizing by default.
+
+## Creating Persistent Volumes
+
+The most straight forward way to create a Persistent Volume is to create a [Persistent Volume Claim object](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims), which will automatically create a corresponding Persistent Volume (PV) for you.
+
+Alternatively, to be able to set more specific parameters on your PV you can first create a [Persistent Volume object](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistent-volumes) and then claim that PV using a Persistent Volume Claim (PVC).
+
+Under the hood, the EBS CSI driver will take care that a corresponding EBS Volume with the correct parameters is created.
+
+The EBS Volume and its data will persist as long as the corresponding PV resource exists. Deleting the resource will also delete the corresponding EBS volume, which means that all stored data will be lost at that point.
+
+## Using Persistent Volumes in a Pod
+
+Once you have a Persistent Volume Claim you can [claim it as a Volume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#claims-as-volumes) in your Pods.
+
+Note that an EBS Volume can only be used by a single Pod at the same time. Thus, the access mode of your PVC can only be `ReadWriteOnce`.
+
+Under the hood the EBS Volume stays detached from your nodes as long as it is not claimed by a Pod. As soon as a Pod claims it, it gets attached to the node that holds the Pod.
+
+## Example
+
+First we create a PVC:
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: my-claim
+spec:
+  accessModes:
+    - ReadWriteOnce
+  storageClassName: ebs-csi
+  resources:
+    requests:
+      storage: 5Gi
+```
+
+Note that the Access Mode while being fixed with EBS still needs to be defined as `ReadWriteOnce` in the manifest.
+
+Further, we need to define the Storage Class `ebs-csi` because it is not the default storage class (default `gp2`).
+
+Now we can create a Pod that uses our PVC:
+
+```yaml
+kind: Pod
+apiVersion: v1
+metadata:
+  name: mypod
+spec:
+  containers:
+    - name: myfrontend
+      image: nginx
+      volumeMounts:
+      - mountPath: "/var/www/html"
+        name: mypd
+  volumes:
+    - name: mypd
+      persistentVolumeClaim:
+        claimName: my-claim
+```
+
+Now we have an NGINX Pod which serves the contents of our EBS Volume.
+
+## Expanding Persistent Volume Claims
+
+Starting with Kubernetes v1.16 CSI Volume can be expanded by simply editing the claim and requesting a larger size. It will trigger an update in the underlying Persistent Volume and EBS Volume (Kubernetes always uses the existing one).
```

Comment:
```suggestion
Starting with Kubernetes v1.16, CSI volumes can be expanded by simply editing the claim (the `PersistentVolumeClaim` resource) and requesting a larger size. This will trigger an update of the associated `PersistentVolume` resource and the underlying EBS volume.
```

I think `(Kubernetes always uses the existing one)` is expected and thus not necessary.